sandbox-apps/ecommerce-webapp/README.md

```diff
@@ -2,108 +2,130 @@
 </details>
 
 <details>
   <summary><a href="https://app.datadoghq.com/containers" target="_blank">Containers List</a></summary>
   
-  ![Container List](https://github.com/DataDog/dpn/blob/master/sandbox-apps/voting-app/static/images/container_list.jpg)
+  ![Container List](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/container_list.jpg)
 </details>
 
 <details>
   <summary><a href="https://app.datadoghq.com/process" target="_blank">Processes List</a></summary>
   
-  ![Process List](https://github.com/DataDog/dpn/blob/master/sandbox-apps/voting-app/static/images/process_list.jpg)
+  ![Process List](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/processes_list.jpg)
 </details>
 
 <details>
   <summary><a href="https://app.datadoghq.com/apm/map?env=dpn-sandbox" target="_blank">Service Map</a></summary>
   
   (make sure to scope to the relevant "env")
 
-  ![Service Map](https://github.com/DataDog/dpn/blob/master/sandbox-apps/voting-app/static/images/service_map.jpg)
+  ![Service Map](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/service_map.jpg)
 </details>
 
 <details>
   <summary><a href="https://app.datadoghq.com/apm/traces?env=dpn-sandbox" target="_blank">Trace Search</a></summary>
   
   (make sure to scope to the relevant "env")
 
-  ![Trace Search](https://github.com/DataDog/dpn/blob/master/sandbox-apps/voting-app/static/images/trace_search.jpg)
+  ![Trace Search](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/trace_search.jpg)
 </details>
 
 <details>
   <summary><a href="https://app.datadoghq.com/logs" target="_blank">Log Explorer</a></summary>
 
-  (you may have to click through the "getting started" flow before you see the logs)
+  (you may have to click through the "getting started" flow before you see the logs, make sure to filter out the service "agent".)
   
-  ![Log Explorer](https://github.com/DataDog/dpn/blob/master/sandbox-apps/voting-app/static/images/log_explorer.jpg)
+  ![Log Explorer](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/log_explorer.jpg)
 </details>
 
 <details>
-  <summary><a href="https://app.datadoghq.com/screen/integration/256/jvm-metrics" target="_blank">JVM Integration Dashboard</a></summary>
+  <summary><a href="https://app.datadoghq.com/rum" target="_blank">Real User Monitoring</a></summary>
+  
+  You will find "storedog-XXX" (where XXX is the name of the host you defined in your setup.env file) in the list of RUM Applications. 
+
+  ![RUM Applications](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/rum_applications.jpg)
+
+  Drill-down into your RUM Application to see performance details on the application
+
+  ![RUM Details](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/rum_applications.jpg)
```

Comment:
```suggestion
  ![RUM Details](https://github.com/DataDog/dpn/blob/master/sandbox-apps/ecommerce-webapp/static/images/rum_details.jpg)
```

think this might be referencing the previous image. i see there is a `rum_details` image instead, which sounds like the correct one if i had to guess?